app/src/main/java/org/mifos/mobile/screens/AboutUsScreen.kt

```diff
@@ -0,0 +1,273 @@
+package org.mifos.mobile.screens
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import org.mifos.mobile.R
+import org.mifos.mobile.theme.AppTheme
+import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.utils.Constants.licenseLink
+import org.mifos.mobile.utils.Constants.sourceCodeLink
+import org.mifos.mobile.utils.Constants.websiteLink
+import java.util.*
+
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+@Preview
```

Comment:
Please remove the @Preview annotation from here and create another function to show a preview of this screen, that function should have a `Preview` suffix. For example, something like: `AboutViewPreview`